install.sh

```diff
@@ -0,0 +1,567 @@
+                echo "Warning: Not enforcing TLS v1.2, this is potentially less secure"
+                _err=$(curl $_retry --silent --show-error --fail --location "$1" --output "$2" 2>&1)
+                _status=$?
+            else
+                _err=$(curl $_retry --proto '=https' --tlsv1.2 --silent --show-error --fail --location "$1" --output "$2" 2>&1)
+                _status=$?
+            fi
+        fi
+        if [ -n "$_err" ]; then
+            echo "$_err" >&2
+            if echo "$_err" | grep -q 404$; then
+                err "installer for platform '$3' not found, this may be unsupported"
+            fi
+        fi
+        return $_status
+    elif [ "$_dld" = wget ]; then
+        if [ "$(wget -V 2>&1 | head -2 | tail -1 | cut -f1 -d" ")" = "BusyBox" ]; then
+            echo "Warning: using the BusyBox version of wget.  Not enforcing strong cipher suites for TLS or TLS v1.2, this is potentially less secure"
+            _err=$(wget "$1" -O "$2" 2>&1)
+            _status=$?
+        else
+            get_ciphersuites_for_wget
+            _ciphersuites="$RETVAL"
+            if [ -n "$_ciphersuites" ]; then
+                _err=$(wget --https-only --secure-protocol=TLSv1_2 --ciphers "$_ciphersuites" "$1" -O "$2" 2>&1)
+                _status=$?
+            else
+                echo "Warning: Not enforcing strong cipher suites for TLS, this is potentially less secure"
+                if ! check_help_for "$3" wget --https-only --secure-protocol; then
+                    echo "Warning: Not enforcing TLS v1.2, this is potentially less secure"
+                    _err=$(wget "$1" -O "$2" 2>&1)
+                    _status=$?
+                else
+                    _err=$(wget --https-only --secure-protocol=TLSv1_2 "$1" -O "$2" 2>&1)
+                    _status=$?
+                fi
+            fi
+        fi
+        if [ -n "$_err" ]; then
+            echo "$_err" >&2
+            if echo "$_err" | grep -q ' 404 Not Found$'; then
+                err "installer for platform '$3' not found, this may be unsupported"
+            fi
+        fi
+        return $_status
+    else
+        err "Unknown downloader" # should not reach here
+    fi
+}
+
+unzip() {
+    local _file="$1"
+    local _dir="$2"
+
+    tar -xzvf "$_file" -C "$_dir"
+}
+
+get_default_install_dir() {
+    [ -z "${XDG_CONFIG_HOME-}" ] && printf %s "${HOME}/.temporalio" || printf %s "${XDG_CONFIG_HOME}/temporalio"
+}
+
+get_install_dir() {
+    local _dir
+    _dir="$(get_default_install_dir)"
+    if [ -n "${TEMPORAL_DIR-}" ]; then
+        _dir="$TEMPORAL_DIR"
+    fi
+    printf %s "$_dir"
+}
+
+install_env() {
+    local _dir="$1"
+
+    cat >"$_dir/env" <<EOL
+#!/bin/sh
+case ":\${PATH}:" in
+    *:"$_dir":*)
+        ;;
+    *)
+        export PATH="$_dir:\$PATH"
+        ;;
+esac
+EOL
+
+    local _source=". $_dir/env"
+    say "To add temporal to PATH update your user profile with the following (ex. ~/.bash_profile):"
```

Comment:
```suggestion
    say "If you are using bash you can add temporal to PATH by appending the following to your user profile (ex. ~/.bash_profile):"
```

Reply:
```suggestion
    say "If you are using bash, you can add Temporal to PATH by appending the following to your user profile (ex. ~/.bash_profile):"
```

Reply:
```suggestion
    say "If you are using bash, you can add temporal to PATH by appending the following to your user profile (e.g. ~/.bash_profile):"
```

the temporal command is lowercase

Reply:
You can have a shell specific message here.